cmd/create/oidcconfig/cmd.go

```diff
@@ -294,23 +294,29 @@ func run(cmd *cobra.Command, _ []string) {
 		r.Reporter.Errorf("%s", err)
 		os.Exit(1)
 	}
-	oidcConfigStrategy.execute(r)
+	oidcConfigId := oidcConfigStrategy.execute(r)
 	if !args.rawFiles {
 		arguments.DisableRegionDeprecationWarning = true // disable region deprecation warning
-		oidcprovider.Cmd.Run(oidcprovider.Cmd, []string{"", mode, oidcConfigInput.IssuerUrl})
+		providerArgs := []string{"", mode, oidcConfigInput.IssuerUrl}
+		if oidcConfigId != "" {
+			providerArgs = append(providerArgs, "--oidc-config-id", oidcConfigId)
+		} else if r.Cluster != nil && r.Cluster.ID() != "" {
+			providerArgs = append(providerArgs, "--cluster", r.Cluster.ID())
+		}
```

Comment:
@hunterkepley Should we provide a clear error here if neither is available? Can that happen?

Reply:
It seems after talking with @gdbranco - that we will always have an OIDC config in this case. I will provide an error though, and remove the cluster ID stuff